src/js/components/Volunteer/Volunteer.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import Navbar from "../Navbar/Navbar";
+import { NavLink } from "react-router-dom";
+import "./volunteer.scss";
+
+class Volunteer extends Component {
+
+
+
+
+    componentWillMount() {
```

Comment:
There is a controversy regarding the call of fetch method in the componentWillMount.
One thing remains, your call is an asynchronous call that will only return when the render happens or later.
Your rendering won't be able to be paused waiting for the data to be displayed.
With this method you have better chances for your component to be empty first up until your data is rendered.
Since you have chosen this path, find a suitable way to grey out the field that are waiting to display data.
Else, you can use the componentDidMount